clouddriver-web/config/clouddriver.yml

```diff
@@ -31,6 +31,7 @@ swagger:
     - .*azure.*
     - .*gce.*
     - .*kubernetes.*
+    - .*dockerRegistry.*
```

Comment:
If there are no new controller endpoints exposed, probably don't need this (yet).